chain/chain/src/test_utils.rs

```diff
@@ -160,16 +160,19 @@ impl RuntimeAdapter for KeyValueRuntime {
         _block_hash: &CryptoHash,
         _receipts: &Vec<Vec<ReceiptTransaction>>,
         transactions: &Vec<SignedTransaction>,
+        generate_storage_proof: Option<()>,
```

Comment:
What's the point of having `Option<()>` instead of `bool`?

Reply:
no point, can change to bool